catalog-info.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: gratibot
```

Comment:
Seems like this is configured for gratibot and not springtrader?

Reply:
Thanks for calling that out--copy/paste error. I just looked it over again and it should be ready now.